### PR TITLE
Kademlia js

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: log/js/src/main/scala/fluence/log/DateFormat.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+package fluence.log
+
+import java.util.Date
+
+import scala.scalajs.js
+
+object DateFormat {
```
Comment (alari) [inline log/js/src/main/scala/fluence/log/DateFormat.scala]: Docs please (explain why it exists).

File: log/jvm/src/main/scala/fluence/log/DateFormat.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fluence.log
+
+import java.text.SimpleDateFormat
+import java.util.Date
+
+object DateFormat {
```
Comment (alari) [inline log/jvm/src/main/scala/fluence/log/DateFormat.scala]: Docs, please.

And maybe a shared interface?

File: node/src/main/scala/fluence/node/MasterNodeApp.scala [hunk 3]
```diff
@@ -72,7 +72,7 @@ object MasterNodeApp extends IOApp {
               keyPair <- Resource.liftF(Configuration.readTendermintKeyPair(masterConf.rootPath))
               kad ← KademliaNode.make[IO, IO.Par](
                 masterConf.kademlia,
-                Ed25519.tendermintAlgo,
+                Ed25519.signAlgo,
```
Comment (alari) [inline node/src/main/scala/fluence/node/MasterNodeApp.scala]: Is it the same?

Comment (DieMyst) [reply node/src/main/scala/fluence/node/MasterNodeApp.scala]: yes

**Pull request merged**